### ISSUE TITLE
Error in generate steps function 

### ISSUE THREAD

**Issue by wisundstrom** [learn-co-curriculum/applying-gradient-descent-lab#10]
the generate steps function is defined as follows:
```def generate_steps(m, b, number_of_steps, movies, learning_rate):```
but then when it is called later the m and b lists are fed into the generate steps function in the wrong order:
```iterations = generate_steps(initial_regression_line['b'], initial_regression_line['m'], 100, scaled_movies, .0001) or [{'m': 'uncomment generate_steps method', 'b': 'uncomment generate_steps method '}]```

This is not on purpose because if you look at the first step in the iterations list generated it is different than when the first step is calculated by itself earlier.

**Comment by mas16** [comment]
Thanks wisundstrom! This has been corrected.
